jobs/octopus-energy-python-django-engineer.html

```diff
@@ -0,0 +1,73 @@
+---
+title: Python / Django Engineer
+company: Octopus Energy
+url: https://octopus.energy
+location: Sydney, Australia
+contract: permanent
+contact:
+    name: James Beith
+    email: talent@octopus.energy
+created: !!timestamp 2019-06-05
+tags:
+  - django
+  - python
+  - sydney
+  - australia
```

Comment:
Please don't include location in tags - these should be the required or desired technical skills.